packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -1423,6 +1423,15 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/tem
  {{- if .Values.prometheus.prometheusSpec.nodeSelector }}
    nodeSelector:
  {{ toYaml .Values.prometheus.prometheusSpec.nodeSelector | indent 4 }}
+@@ -226,7 +229,7 @@
+ {{- end }}
+ {{- if .Values.prometheus.prometheusSpec.containers }}
+   containers:
+-{{ toYaml .Values.prometheus.prometheusSpec.containers | indent 4 }}
++{{ tpl .Values.prometheus.prometheusSpec.containers $ | indent 4 }}
```

Comment:
Updated since we pass in a template that needs to be rendered by Helm in order to support private registry stuff with the containers we are inserting.